src/control/cmd/dmg/acl.go

```diff
@@ -44,6 +44,12 @@ func readACLFile(aclFile string) ([]string, error) {
 	return parseACL(file)
 }
 
+// isACLFileComment checks whether the line is formatted as a comment for an
+// ACL file.
+func isACLFileComment(line string) bool {
+	return strings.HasPrefix(line, "#")
```

Comment:
This seems a bit fragile... What if the line looks like this:
```
  # foo bar
```

You could run the line through strings.TrimSpace() first. Could also use a regex but that might be overkill.

Reply:
Oh, sorry. I see that the line is already going through TrimSpace before it's checked here.